common/app/views/support/GetClasses.scala

```diff
@@ -22,6 +22,8 @@ object GetClasses {
       ("js-fc-item", true),
       ("fc-item--pillar-" + item.pillar.nameOrDefault, experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
       ("fc-item--type-" + item.designType.nameOrDefault, experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
+      ("fc-item--type-" + item.cardStyle.toneString, experiments.ActiveExperiments.isParticipating(experiments.Garnett) && item.cardStyle == SpecialReport),
+      ("fc-item--type-guardian-labs", experiments.ActiveExperiments.isParticipating(experiments.Garnett) && item.branding.exists(_.isPaid)),
```

Comment:
@blongden73 will that be enough for you?

Reply:
how about `fc-item--special-report` as a bonus class, rather than `fc-item--type-special-report`?

Reply:
Yes, sounds good 👍